[2024-09-19T17:34:12.607+0000] {taskinstance.py:1171} INFO - Dependencies all met for <TaskInstance: dag_with_python_operator_v07.greet scheduled__2024-09-18T00:00:00+00:00 [queued]>
[2024-09-19T17:34:12.612+0000] {taskinstance.py:1171} INFO - Dependencies all met for <TaskInstance: dag_with_python_operator_v07.greet scheduled__2024-09-18T00:00:00+00:00 [queued]>
[2024-09-19T17:34:12.612+0000] {taskinstance.py:1368} INFO - 
--------------------------------------------------------------------------------
[2024-09-19T17:34:12.612+0000] {taskinstance.py:1369} INFO - Starting attempt 1 of 6
[2024-09-19T17:34:12.612+0000] {taskinstance.py:1370} INFO - 
--------------------------------------------------------------------------------
[2024-09-19T17:34:12.619+0000] {taskinstance.py:1389} INFO - Executing <Task(PythonOperator): greet> on 2024-09-18 00:00:00+00:00
[2024-09-19T17:34:12.621+0000] {standard_task_runner.py:52} INFO - Started process 7789 to run task
[2024-09-19T17:34:12.623+0000] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'dag_with_python_operator_v07', 'greet', 'scheduled__2024-09-18T00:00:00+00:00', '--job-id', '50', '--raw', '--subdir', 'DAGS_FOLDER/create_dag_with_python_operator.py', '--cfg-path', '/tmp/tmptgjscsi4', '--error-file', '/tmp/tmpa9w_rawj']
[2024-09-19T17:34:12.624+0000] {standard_task_runner.py:80} INFO - Job 50: Subtask greet
[2024-09-19T17:34:12.660+0000] {task_command.py:371} INFO - Running <TaskInstance: dag_with_python_operator_v07.greet scheduled__2024-09-18T00:00:00+00:00 [running]> on host a83b02c526f7
[2024-09-19T17:34:12.696+0000] {taskinstance.py:1583} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=faruk3d
AIRFLOW_CTX_DAG_ID=dag_with_python_operator_v07
AIRFLOW_CTX_TASK_ID=greet
AIRFLOW_CTX_EXECUTION_DATE=2024-09-18T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2024-09-18T00:00:00+00:00
[2024-09-19T17:34:12.697+0000] {taskinstance.py:1902} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 171, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 189, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
TypeError: greet() missing 1 required positional argument: 'age'
[2024-09-19T17:34:12.762+0000] {taskinstance.py:1412} INFO - Marking task as UP_FOR_RETRY. dag_id=dag_with_python_operator_v07, task_id=greet, execution_date=20240918T000000, start_date=20240919T173412, end_date=20240919T173412
[2024-09-19T17:34:12.767+0000] {standard_task_runner.py:97} ERROR - Failed to execute job 50 for task greet (greet() missing 1 required positional argument: 'age'; 7789)
[2024-09-19T17:34:12.804+0000] {local_task_job.py:156} INFO - Task exited with return code 1
[2024-09-19T17:34:12.817+0000] {local_task_job.py:279} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-09-19T18:41:23.184+0000] {taskinstance.py:1171} INFO - Dependencies all met for <TaskInstance: dag_with_python_operator_v07.greet scheduled__2024-09-18T00:00:00+00:00 [queued]>
[2024-09-19T18:41:23.188+0000] {taskinstance.py:1171} INFO - Dependencies all met for <TaskInstance: dag_with_python_operator_v07.greet scheduled__2024-09-18T00:00:00+00:00 [queued]>
[2024-09-19T18:41:23.188+0000] {taskinstance.py:1368} INFO - 
--------------------------------------------------------------------------------
[2024-09-19T18:41:23.188+0000] {taskinstance.py:1369} INFO - Starting attempt 1 of 6
[2024-09-19T18:41:23.188+0000] {taskinstance.py:1370} INFO - 
--------------------------------------------------------------------------------
[2024-09-19T18:41:23.193+0000] {taskinstance.py:1389} INFO - Executing <Task(PythonOperator): greet> on 2024-09-18 00:00:00+00:00
[2024-09-19T18:41:23.195+0000] {standard_task_runner.py:52} INFO - Started process 12455 to run task
[2024-09-19T18:41:23.196+0000] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'dag_with_python_operator_v07', 'greet', 'scheduled__2024-09-18T00:00:00+00:00', '--job-id', '57', '--raw', '--subdir', 'DAGS_FOLDER/create_dag_with_python_operator.py', '--cfg-path', '/tmp/tmp351ucsyg', '--error-file', '/tmp/tmpr_tn_eqx']
[2024-09-19T18:41:23.197+0000] {standard_task_runner.py:80} INFO - Job 57: Subtask greet
[2024-09-19T18:41:23.220+0000] {task_command.py:371} INFO - Running <TaskInstance: dag_with_python_operator_v07.greet scheduled__2024-09-18T00:00:00+00:00 [running]> on host a83b02c526f7
[2024-09-19T18:41:23.247+0000] {taskinstance.py:1583} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=faruk3d
AIRFLOW_CTX_DAG_ID=dag_with_python_operator_v07
AIRFLOW_CTX_TASK_ID=greet
AIRFLOW_CTX_EXECUTION_DATE=2024-09-18T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2024-09-18T00:00:00+00:00
[2024-09-19T18:41:23.318+0000] {logging_mixin.py:115} INFO - Hello World! My name is Jerry Simp, and I am 20 years old.
[2024-09-19T18:41:23.318+0000] {python.py:173} INFO - Done. Returned value was: None
[2024-09-19T18:41:23.323+0000] {taskinstance.py:1412} INFO - Marking task as SUCCESS. dag_id=dag_with_python_operator_v07, task_id=greet, execution_date=20240918T000000, start_date=20240919T184123, end_date=20240919T184123
[2024-09-19T18:41:23.334+0000] {local_task_job.py:156} INFO - Task exited with return code 0
[2024-09-19T18:41:23.347+0000] {local_task_job.py:279} INFO - 0 downstream tasks scheduled from follow-on schedule check
